RealmSwift/List.swift

```diff
@@ -433,6 +433,10 @@ public final class List<Element: RealmCollectionValue>: RLMSwiftCollectionBase {
         return Element._rlmArray()
     }
 
+    @objc public override class func backingCollectionType() -> AnyClass {
```

Comment:
should this not be `internal`?

Reply:
Ideally it would be but it can't be since it's overriding a base class method. It should be underscore-prefixed though.

Reply:
Ahh I see. Does it require any doc? I see its missing `:nodoc:`